File: HealthSysHub.Web.Utility/Models/ApplicationUser.cs
namespace HealthSysHub.Web.Utility.Models
{
    public class ApplicationUser
    {
        public Guid? Id { get; set; }
        public string? FullName { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public Guid? RoleId { get; set; }
        public string? RoleName { get; set; }
        public Guid? HospitalId { get; set; }
        public Guid? StaffId { get; set; }
        public string? HospitalName { get; set; }
        public string? Designation { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/applicationuser.ts
export interface IApplicationUser {
    id?: string;                   // Guid? in C#
    fullName?: string;             // string? in C#
    firstName?: string;            // string? in C#
    lastName?: string;             // string? in C#
    email?: string;                // string? in C#
    phone?: string;                // string? in C#
    roleId?: string;               // Guid? in C#
    roleName?: string;             // string? in C#
    hospitalId?: string;           // Guid? in C#
    staffId?: string;              // Guid? in C#
    labId?: string;                // Guid? in C# (new property)
    pharmacyId?: string;           // Guid? in C# (new property)
    hospitalName?: string;         // string? in C#
    designation?: string;          // string? in C#
}